### PR TITLE
feat: add code sample and test for concatenating two input videos

### PR DESCRIPTION
**Pull request by irataxy**
b:197546828

### CODE REVIEW

File: media/transcoder/src/create_job_with_concatenated_inputs.php [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+ * @param float  $endTimeInput1 End time, in fractional seconds, relative to the first input video timeline.
+ * @param string $input2Uri Uri of the second video in the Cloud Storage bucket.
+ * @param float  $startTimeInput2 Start time, in fractional seconds, relative to the second input video timeline.
+ * @param float  $endTimeInput2 End time, in fractional seconds, relative to the second input video timeline.
+ * @param string $outputUri Uri of the video output folder in the Cloud Storage bucket.
+ */
+function create_job_with_concatenated_inputs($projectId, $location, $input1Uri, $startTimeInput1, $endTimeInput1, $input2Uri, $startTimeInput2, $endTimeInput2, $outputUri)
+{
+    $startTimeInput1Sec = (int)floor(abs($startTimeInput1));
+    $startTimeInput1Nanos = (int)(1000000000 * bcsub(abs($startTimeInput1), floor(abs($startTimeInput1)), 4));
```
Comment (irataxy) [inline media/transcoder/src/create_job_with_concatenated_inputs.php:55]: Do I need to add BCMath to `composer.json`?

Comment (bshaffer) [reply media/transcoder/src/create_job_with_concatenated_inputs.php:55]: yes, that's a good idea. Just `ext-bcmath` under the required section

**Review by bshaffer — COMMENTED** [review]
The linter needs you to fix the following things:

```diff
-    $startTimeInput1Sec = (int)floor(abs($startTimeInput1));
-    $startTimeInput1Nanos = (int)(1000000000 * bcsub(abs($startTimeInput1), floor(abs($startTimeInput1)), 4));
-    $endTimeInput1Sec = (int)floor(abs($endTimeInput1));
-    $endTimeInput1Nanos = (int)(1000000000 * bcsub(abs($endTimeInput1), floor(abs($endTimeInput1)), 4));
+    $startTimeInput1Sec = (int) floor(abs($startTimeInput1));
+    $startTimeInput1Nanos = (int) (1000000000 * bcsub(abs($startTimeInput1), floor(abs($startTimeInput1)), 4));
+    $endTimeInput1Sec = (int) floor(abs($endTimeInput1));
+    $endTimeInput1Nanos = (int) (1000000000 * bcsub(abs($endTimeInput1), floor(abs($endTimeInput1)), 4));

-    $startTimeInput2Sec = (int)floor(abs($startTimeInput2));
-    $startTimeInput2Nanos = (int)(1000000000 * bcsub(abs($startTimeInput2), floor(abs($startTimeInput2)), 4));
-    $endTimeInput2Sec = (int)floor(abs($endTimeInput2));
-    $endTimeInput2Nanos = (int)(1000000000 * bcsub(abs($endTimeInput2), floor(abs($endTimeInput2)), 4));
+    $startTimeInput2Sec = (int) floor(abs($startTimeInput2));
+    $startTimeInput2Nanos = (int) (1000000000 * bcsub(abs($startTimeInput2), floor(abs($startTimeInput2)), 4));
+    $endTimeInput2Sec = (int) floor(abs($endTimeInput2));
+    $endTimeInput2Nanos = (int) (1000000000 * bcsub(abs($endTimeInput2), floor(abs($endTimeInput2)), 4));
```

File: media/transcoder/README [hunk 2]
```diff
@@ -50,7 +50,21 @@ See the [Transcoder Documentation](https://cloud.google.com/transcoder/docs/) fo
 
 ## Troubleshooting
 
-* See the [Troubleshooting guide](https://cloud.google.com/transcoder/docs/troubleshooting) for more information.
+### bcmath extension missing
+
+If you see an error like this:
+
+```
+PHP Fatal error:  Uncaught Error: Call to undefined function Google\Protobuf\Internal\bcsub() in /usr/local/google/home/user/github/GoogleCloudPlatform/php-docs-samples/media/transcoder/vendor/google/protobuf/src/Google/Protobuf/Internal/Message.php:986
```
Comment (bshaffer) [inline media/transcoder/README:58]: Your error message includes your local path to the files, and is way too specific.
```suggestion
PHP Fatal error:  Uncaught Error: Call to undefined function Google\Protobuf\Internal\bcsub()
```

**Review by bshaffer — CHANGES_REQUESTED** [review]

File: media/transcoder/README [hunk 3]
```diff
@@ -50,7 +50,21 @@ See the [Transcoder Documentation](https://cloud.google.com/transcoder/docs/) fo
...
+```
+
+You may need to install the bcmath PHP extension.
+e.g. (may depend on your php version)
+```
+$ sudo apt-get install php7.3-bcmath
+```
```
Comment (bshaffer) [inline media/transcoder/README:65]: as mentioned before, these samples _all_ require BC-math. So change this message to simply "you need to install the BC-Math extension" (no instructions necessary, it varies per platform and PHP version, so writing out installation instructions will do more harm than good), and add "ext-bcmath" to the required section of `composer.json`

Comment (irataxy) [reply media/transcoder/README]: Added

File: media/transcoder/src/create_job_with_concatenated_inputs.php (hunk 1 above)
Comment (irataxy) [reply media/transcoder/src/create_job_with_concatenated_inputs.php]: Done

**Review by bshaffer — APPROVED** [review]

**Pull request merged**